src/main/java/org/folio/circulation/domain/policy/lostitem/LostItemPolicy.java

```diff
@@ -1,51 +1,73 @@
 package org.folio.circulation.domain.policy.lostitem;
 
+import static java.util.Optional.ofNullable;
+import static org.folio.circulation.domain.policy.lostitem.ChargeAmountType.SET_COST;
+import static org.folio.circulation.domain.policy.lostitem.ChargeAmountType.forValue;
 import static org.folio.circulation.support.JsonPropertyFetcher.getBigDecimalProperty;
 import static org.folio.circulation.support.JsonPropertyFetcher.getBooleanProperty;
 import static org.folio.circulation.support.JsonPropertyFetcher.getObjectProperty;
 import static org.folio.circulation.support.JsonPropertyFetcher.getProperty;
 
 import java.math.BigDecimal;
+import java.util.Optional;
 
 import org.folio.circulation.domain.policy.Policy;
 
 import io.vertx.core.json.JsonObject;
 
 public class LostItemPolicy extends Policy {
-  private final ChargeAmount chargeAmountItem;
+  private final ItemFee setCostChargeFee;
```

Comment:
Could this be shortened to `setCostFee`? What is the significance of `charge` in this context?

```suggestion
  private final ItemFee setCostFee;
```